.github/workflows/mtags-auto-release.yml

```
@@ -0,0 +1,31 @@
+name: "Mtags auto release"
+on:
+  workflow_dispatch:
+    inputs:
+      scala_version:
+        description: "Scala Version"
+        required: true
+      metals_version:
+        description: "Metals Version"
+        required: true
+        default: "v0.10.9"
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: olafurpg/setup-scala@v13
+      - uses: coursier/cache-action@v6.3
+      - name: Test Dyn
```

Since workflow is oriented towards publishing, maybe
```suggestion
      - name: Publish Mtags
```

---

Changed it a bit differently